api/src/team/domain/usecases/update-certification-center-membership-last-accessed-at.usecase.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Update the last accessed at date of a certification center membership
+ * @param {Object} params
+ * @param {string} params.userId
+ * @param {string} params.certificationCenterId
+ * @param {CertificationCenterMembershipRepository} params.certificationCenterMembershipRepository
+ */
+
+const updateCertificationCenterMembershipLastAccessedAt = async function ({
```

Comment:
Suppression de la paraphrase inutile _« Update the last accessed at date of a certification center membership »_ : 

```suggestion
/**
 * @param {Object} params
 * @param {string} params.userId
 * @param {string} params.certificationCenterId
 * @param {CertificationCenterMembershipRepository} params.certificationCenterMembershipRepository
 */
const updateCertificationCenterMembershipLastAccessedAt = async function ({
```